### ISSUE TITLE
Suggestion: Meta-Data Skip

### ISSUE THREAD

**Issue by ArcaneFuror** [deepjyoti30/ytmdl#74]
I've been playing around with the script, and I like what it can do. I especially love the --list feature

However, selecting the meta-data each time is a bit annoying, especially since a lot of the cases are not showing the correct artists despite passing the tags via the various arguments and checking on itunes and the like for availability. Most results are only slightly relevant in title on the meta side of things, not the artists. This is fine as I'm just after the music.

In short, it'd be nice to be able to have the option to skip the segment that adds the niceties like cover art and just saves the audio, perhaps just defaulting the name using the search term used originally via the command line or the passed list. Which would allow it to pair nicely with the -q flag since Youtube is quite reliable on first attempts, allowing for quick and efficient execution.

**Comment by wpgrant** [comment]
+1 for this suggestion. This might be a separate situation but even if you use it in non-quiet mode you can't request skipping the metadata part. (You can still find the download in the .cache directory.

A counter-argument is that the main point of this is to add the metadata, but I'd still like to be able to use the same script for both use cases.

**Comment by deepjyoti30** [comment]
Hey guys, added support for skipping the meta, check #adc705537422ca3e1438004fc92f8bd001afbbae 

In case you face any  bugs, report back here.

**Comment by phmassa** [comment]
On Windows10 with Debian 10 WSL, edition work fine for me

**Comment by deepjyoti30** [comment]
Added in latest release.
